topdown/time.go

```diff
@@ -19,6 +19,28 @@ import (
 var tzCache map[string]*time.Location
 var tzCacheMutex *sync.Mutex
 
+// -9223372036854775808ns, int64 wraparound point
+var minDateAllowedForNsConversion = time.Date(1677, 9, 21, 0, 12, 43, 145224192, time.UTC)
```

Comment:
This is quite opaque, is there a simple way to express this using `math.MaxInt64`? 🤔